app/javascript/react/containers/ReviewContainer.js

```diff
@@ -0,0 +1,101 @@
+import React, {Component } from "react";
+import ReviewTile from '../components/ReviewTile'
+
+class ReviewContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      clickCounts: null
+    }
+    this.handleClick = this.handleClick.bind(this)
+  }
+
+    handleClick(event, reviewId) {
```

Comment:
no changes needed, but fyi that when you see a lot of repetitive code, there's usually a way to refactor for simplicity :)